<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-05-10T18:07:22 IST" name="Suite" finished-at="2024-05-10T18:08:04 IST" duration-ms="41835">
    <groups>
    </groups>
    <test started-at="2024-05-10T18:07:22 IST" name="Test" finished-at="2024-05-10T18:08:04 IST" duration-ms="41835">
      <class name="com.comcast.crm.orgTest.CreateOrganizationTest">
        <test-method is-config="true" signature="configBS()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:07:22 IST" name="configBS" finished-at="2024-05-10T18:07:22 IST" duration-ms="75" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBS -->
        <test-method is-config="true" signature="configBC()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:07:22 IST" name="configBC" finished-at="2024-05-10T18:07:33 IST" duration-ms="10529" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBC -->
        <test-method is-config="true" signature="configBM()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:07:33 IST" name="configBM" finished-at="2024-05-10T18:07:38 IST" duration-ms="5952" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBM -->
        <test-method signature="createOrganizationTest()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:07:38 IST" name="createOrganizationTest" finished-at="2024-05-10T18:07:59 IST" duration-ms="20095" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [  Facebook_2305] but found [Facebook_2305]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [  Facebook_2305] but found [Facebook_2305]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.comcast.crm.orgTest.CreateOrganizationTest.createOrganizationTest(CreateOrganizationTest.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createOrganizationTest -->
        <test-method is-config="true" signature="configAM()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:07:59 IST" name="configAM" finished-at="2024-05-10T18:08:03 IST" duration-ms="4133" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAM -->
        <test-method is-config="true" signature="configAC()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:08:03 IST" name="configAC" finished-at="2024-05-10T18:08:04 IST" duration-ms="720" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAC -->
        <test-method is-config="true" signature="configAS()[pri:0, instance:com.comcast.crm.orgTest.CreateOrganizationTest@712625fd]" started-at="2024-05-10T18:08:04 IST" name="configAS" finished-at="2024-05-10T18:08:07 IST" duration-ms="3004" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAS -->
      </class> <!-- com.comcast.crm.orgTest.CreateOrganizationTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
